ci/environment-py38-upstream-master.yml

```diff
@@ -0,0 +1,15 @@
+name: test_env_xgcm
+dependencies:
+  - python=3.8
+  - xarray
+  - dask
+  - numpy
+  - numba
+  - pytest
+  - future
+  - pip
+  - black
+  - pip:
+    - docrep<=0.2.7
+    - codecov
+    - pytest-cov
```

Comment:
I must admit I'm confused about this environment and don't understand why its' here.

Reply:
It shouldnt....